### PR TITLE
 Fix views overwriting

### PR DESCRIPTION
**Pull request by rmberrios**
### Fixed

- Fixed views overwriting existing array

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+### Fixed
+
+- Fixed views overwriting existing array
```
Comment (armonge) [inline CHANGELOG.md]: what was the issue?

Comment (rmberrios) [reply CHANGELOG.md]: instead of merging repeated path, it was only saving the last one

File: test/utils.ts [hunk 2]
```diff
@@ -33,6 +33,7 @@ function loadInteraction(name: interactionNames) {
 export function configurationToExecute() {
   const excelInteraction = loadInteraction("Excel");
   const openDocumentInteraction = loadInteraction("OpenDocument");
+  const LibreOfficeInteraction = loadInteraction("LibreOffice");
```
Comment (armonge) [inline test/utils.ts]: Actually,  ODS and FODS are both open document, the difference is that fods is a Flat Open Document spreadhseet, basically an uncompressed ODS

Comment (rmberrios) [reply test/utils.ts]: I renamed them

**Pull request merged**